dependencies/pom.xml

```diff
@@ -23,6 +23,11 @@
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
what other dependencies does this pull in?

Reply:
Looks like this adds no transitive compile scope dependencies.

Reply:
Yes.

```
$ mvn dependency:tree
...
[INFO] +- com.google.cloud.tools.opensource:dependency-lister:jar:1.0-SNAPSHOT:compile
[INFO] |  +- commons-cli:commons-cli:jar:1.4:compile
[INFO] |  +- org.apache.bcel:bcel:jar:6.2:compile
[INFO] |  +- org.apache.maven:maven-resolver-provider:jar:3.5.4:compile
...
```